homeassistant/components/advantage_air/climate.py

```diff
@@ -138,6 +151,15 @@ def fan_mode(self) -> str | None:
         """Return the current fan modes."""
         return ADVANTAGE_AIR_FAN_MODES.get(self._ac["fan"])
 
+    @property
+    def preset_mode(self) -> str:
+        """Return the current preset mode."""
+        if self._ac.get(ADVANTAGE_AIR_MYAUTO_ENABLED):
+            return ADVANTAGE_AIR_MYAUTO
+        if self._ac.get(ADVANTAGE_AIR_MYTEMP_ENABLED):
+            return ADVANTAGE_AIR_MYTEMP
+        return ADVANTAGE_AIR_MYZONE
```

Comment:
When a preset is active there will still be an HVAC mode selected and active in the state. I don't think we should have myauto both be a preset and an HVAC mode.

Does one of these three my-options need to be selected always, or can neither of them be active?

Reply:
The system is always in one of the three presets.

"myauto" also happens to be the string you have to send to put the system into its "auto" HVAC mode.